### PR TITLE
add filterButton and basic layout and styling

### PR DESCRIPTION
**Pull request by NirachaMarchett**

### CODE REVIEW

**Comment by Frank-Schaeske** [comment]
Hi Niracha, 

when I open the preview I get the following error messages in the console:

Errors

I don't know what they mean but I guess you should check your code again.

**Comment by NirachaMarchett** [comment]
hey, thank you for reviewing my code, I checked my console and no error messages shown. If you can still see the error messages please let me know :)

File: component/FilterButton/FilterButoon.styled.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { styled } from "styled-components";
```
Comment (Kheirah) [inline component/FilterButton/FilterButoon.styled.js:1]:
```suggestion
import styled from "styled-components";
```
`styled` is a default export.

File: component/GroceriesList/index.js [hunk 2]
```diff
@@ -8,30 +8,77 @@ import {
...
+                </p>
+              </StyledDetailSection>
 
-            <StyledRemainingDays className="remainingDays">
-              Expire in: {calculateDaysRemaining(expirationDate)} days
-            </StyledRemainingDays>
-          </StyledListContainer>
-        );
-      })}
+              <StyledRemainingDays className="remainingDays">
```
Comment (Kheirah) [inline component/GroceriesList/index.js:75]: The classes in `className` don't do anything. You may remove them. We're styling with styled-components anyway.

File: component/GroceriesList/index.js [hunk 3]
```diff
@@ -8,30 +8,77 @@ import {
...
+                </StyledDetailSection>
+                <StyledRemainingDays className="remainingDays">
+                  Expire in: {calculateDaysRemaining(expirationDate)} days
+                </StyledRemainingDays>
+              </StyledListContainer>
+            )
+          )}
+        </StyledList>
+      );
+    }
```
Comment (Kheirah) [inline component/GroceriesList/index.js:43]: You are accidentally using JSX within the filter function. If you look closely, you'll see that the filter function extends to line 52. That's where it ends. Until then, the filter function only needs to know whether or not to include an item.

**Review by Kheirah — CHANGES_REQUESTED** [review]
Nicely done.

See if you can remove the JSX from your filter function. That's the main issue. Then remove all `classNames` as well.

File: component/GroceriesList/index.js (hunk 3 above)
Comment (NirachaMarchett) [reply component/GroceriesList/index.js]: Hey, thank you so much for review my code and very helpful feedback. I pushed it again and did what you suggested :)

**Comment by Bilal4317** [comment]
now it looks right.

**Pull request merged**